bigquery/google/cloud/bigquery/client.py

```diff
@@ -138,9 +138,12 @@ def list_projects(self, max_results=None, page_token=None,
                             passed, defaults to a value set by the API.
 
         :type page_token: str
-        :param page_token: opaque marker for the next "page" of projects. If
-                           not passed, the API will return the first page of
-                           projects.
+        :param page_token: (Optional) Token representing a cursor into the
+                           projects.  If not passed, the API will return
+                           the first page of projects. The token marks the
+                           beginning of the iterator to be returned and the
+                           value of the page_token can be accessed at
+                           next_page_token of the HTTPiterator.
```

Comment:
Ditto on backticks for ``next_page_token``.

Nit. `HTTPIterator`. Also, you could make this a link to the class.